src/ui/components/AssetLink/AssetLink.tsx

```diff
@@ -3,10 +3,38 @@ import React from 'react';
 import { usePreferences } from 'src/ui/features/preferences';
 import { openInNewWindow } from 'src/ui/shared/openInNewWindow';
 import { TextAnchor } from 'src/ui/ui-kit/TextAnchor';
+import { TextLink } from 'src/ui/ui-kit/TextLink';
 
 export function AssetLink({
   asset,
   title,
+}: {
+  asset: Pick<Asset, 'symbol' | 'name' | 'asset_code'>;
+  title?: string;
+}) {
+  const { preferences } = usePreferences();
+  const content = title || asset.symbol || asset.name;
+  if (preferences?.testnetMode?.on) {
+    return content;
```

Comment:
Does it make sense to apply the same style when we're in `testnetMode`? I mean `overflow: hidden`, etc

Reply:
Hm. Makes sense

Reply:
But here I just copied previous implementation. Let's keep it this way and fix if needed in the future separately